main.tf

```diff
@@ -34,7 +44,15 @@ module "obs_staging_vpc" {
   xchain_settings = {
     enabled      = true
     docker_image = local.xchain_indexer_staging_docker_image
-    config       = "staging"
+    config       = "config"
+    json_config = jsonencode({
+      omni_config = {
+        rpc_addr = local.omni_staging_rpc
+      }
+      external_chains = [
+        for x_chain in local.external_chains_staging : x_chain
```

Comment:
IMO we can just declare them both here instead of using locals. because we don't use the locals other places, right?

Reply:
We are using on in multiple places atm: omni_config.rpc_addr. I like the approach to group them on top with other config vars such as the url for ws connection, think it makes the config slightly more readable. Not super opinionated on it though..